project/rroykuo/README.md

```diff
@@ -0,0 +1,57 @@
+# Clip your arbitrary polygons
+
+## Basic Information
+Clipping polygons means using a polygon to clip a subject one. Only intersection parts of both would be retain. This function would be used to image processing or clipping a image into certain shape.
+
+> ![](https://i.imgur.com/jugvja9.png)
+>
+> **Clipping of polygons**
+
+## Problem to solve
+Some algorithms to solve clipping polygons might be fast but have some limit of input (ex. polygons have to be convex). Greiner–Hormann clipping algorithm can be fast and easy than other algorithm. Furthermore, it doesn't limit the shape of polygons. Therefore, I decide to reproduce the polygon clipping function with this algorithm (Greiner–Hormann clipping algorithm) in C++. Both clip and subject polygons could be concave or convex.
```

Comment:
How would you represent holes?

Reply:
I would render original two polygons just with edge. After displaying overlapping part with color, it would represent holes.

Reply:
That is the island format.  Would you handle the cutline format too?

Reply:
I couldn't differentiate island and cutline types. All in all, I would try to display the visual difference before and after the process of clipping to user.